packages/css/src/components/UI.css

```diff
@@ -0,0 +1,64 @@
+/* Headings */
+
+.UI--Heading1,
+.UI--Heading2,
+.UI--Heading3,
+.UI--Heading4,
+.UI--Heading5,
+.UI--Heading6 {
```

Comment:
We have `Heading--x`, but also `UI--HeadingX`. Maybe `UI--Heading-X` for the sake of consistency? And/or changing the  `Heading--x` for  `Heading-x`. 

When using the React comps this doesn't matter much, but with the classes directly consistency is important IMO

Reply:
Yes, nice catch 👍 
Will use the `UI--Heading-X` pattern, even though I had to do some weird stuff on the UI component like
```
+    as={`h${as}`}
+    type={`Heading-${as}`}
```
😄